conda.recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
+npm install .
+npm run build
+python setup.py install
+python -m nbpresent.install --force --enable
```

Comment:
Thanks for the link!

The install `nbpresent.install` actually needs to be run on install.

I also had to add `--user` to allow it to work on my linux machine. @damianavila in your experience, where do most people install extensions (i.e. global/user)? Is there an expectation that the global `/share` location is really shared by every user/app?